src/sentry/models/grouphistory.py

```diff
@@ -258,9 +259,9 @@ def get_prev_history(group, status):
 def record_group_history_from_activity_type(
     group: "Group",
     activity_type: int,
-    actor: Union["User", "Team"] | None = None,
+    actor: Union["RpcUser", "User", "Team"] | None = None,
```

Comment:
Down below we call this code for `actor`:
https://github.com/getsentry/sentry/blob/e0ad1e30924b5538264c84a5e7525ef222a4c702/src/sentry/models/grouphistory.py#L280-L283

https://github.com/getsentry/sentry/blob/e0ad1e30924b5538264c84a5e7525ef222a4c702/src/sentry/models/grouphistory.py#L294-L299